Gemfile

```diff
@@ -59,6 +59,7 @@ gem "compass-rails",  "2.0.5"
 gem "sass-rails",     "5.0.4"
 gem "autoprefixer-rails", "6.3.6.2"
 gem "bootstrap-switch-rails", "3.3.3"
+gem "font-awesome-rails", "4.4.0.0"
```

Comment:
Default use Bootstrap's glyphicons:

![bootstrap-markdown-default](https://cloud.githubusercontent.com/assets/3429231/16557556/e66dac22-41e0-11e6-9ff8-60cc9f5568cb.png)
 
If it's ok with you then I change.